test/integration/targets/ansible_runner_smoketests/files/playbook_example1.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+import json
+import os
+import sys
+import ansible_runner
+
+
+PLAYBOOK = '''
+- hosts: localhost
+  gather_facts: False
+  tasks:
+    - set_fact:
+        foo: bar
+'''
+
+# the first positional arg should be where the artifacts live
+output_dir = sys.argv[1]
+
+invdir = os.path.join(output_dir, 'inventory')
+if not os.path.isdir(invdir):
+    os.makedirs(invdir)
+with open(os.path.join(invdir, 'hosts'), 'w') as f:
+    f.write('localhost\n')
+pbfile = os.path.join(output_dir, 'test.yml')
+with open(pbfile, 'w') as f:
+    f.write(PLAYBOOK)
+
+r = ansible_runner.run(private_data_dir=output_dir, playbook='test.yml')
+
+data = {
```

Comment:
This duplicates previous module converter, it makes sense to put it in a shared utils module

Reply:
These are standalone scripts that are executed inside the runner container, so I was avoiding utils based purely on the fact that i didn't want to ship more than one file over to the container for each test. I could theoretically zip/tar up a couple things on the control node and send that over to the container instead, but that's more work that I wanted to do upfront.